# EdgeDetection.srv
sensor_msgs/Image color
sensor_msgs/Image depth
---
sensor_msgs/Image overlay
visualization_msgs/Marker marker
